lib/dialects/postgres/data-types.js

```diff
@@ -87,6 +88,33 @@ module.exports = BaseTypes => {
 
   BaseTypes.DATEONLY.types.postgres = ['date'];
 
+  function INTERVAL(...args) {
+    if (!(this instanceof INTERVAL)) return new INTERVAL(...args);
+    BaseTypes.INTERVAL.apply(this, args);
+  }
+
+  inherits(INTERVAL, BaseTypes.INTERVAL);
+
+  INTERVAL.parse = function parse(value) {
+    return moment.duration(value);
```

Comment:
Curious, why not use `parse` from `postgres-interval`? https://github.com/bendrucker/postgres-interval/blob/f58ec4bb28d67fe1f9c9a1ad4ab8066c26de24bc/index.js#L113